src/igvfd/upgrade/sample.py

```diff
@@ -248,3 +248,20 @@ def in_vitro_system_11_12(value, system):
     if 'time_post_factors_introduction_units' in value:
         value['time_post_change_units'] = value['time_post_factors_introduction_units']
         del value['time_post_factors_introduction_units']
+
+
+@upgrade_step('whole_organism', '14', '15')
```

Comment:
was that missing? why is it added in this PR?

Reply:
I just noticed whole_organism.py was created with this single upgrade: https://github.com/IGVF-DACC/igvfd/pull/470/files#diff-acbed1fca8facc3ec3a83224b0d74c38c268cd5635a656c8437022c1bdcdd389 and decided to just move it to the shared upgrade file sample.py